aim/web/ui/src/config/analytics/analyticsKeysMap.ts

```diff
@@ -279,6 +279,9 @@ export const ANALYTICS_EVENT_KEYS = {
         deleteRun: '[RunDetail] [Settings] Delete run',
         archiveRun: '[RunDetail] [Settings] Archive run',
       },
+      logs: {
+        tabView: '[RunDetail] [Logs] Tab view',
+      },
```

Comment:
doesn't need to add some events for 'load-more', 'live-update' ?

Reply:
I think we are tracking only data which manually triggered by users.